themes/default/content/blog/announcing-public-preview-update-plans/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Announcing the public preview of Update Plans"
+date: 2022-02-08 # TODO update this
+meta_desc: Update Plans enable you to see and confirm the updates that will be made to your infrastructure and then apply those updates at a later time.
+meta_image: update_plans_preview.png
+authors:
+    - fraser-waters
+tags:
+    - features
+---
+
+## Announcing the public preview of Update Plans
```

Comment:
You should drop this header to avoid
<img width="719" alt="image" src="https://user-images.githubusercontent.com/1454008/152971488-bd67c792-84db-4679-9289-e8187d9c4714.png">